CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## Next
+
+### Changed
+
+- [CL-2168] Added support for webp images.
```

Comment:
I think it would be good to say something about the image formats that are taken away. And I would list all formats that we support as of this PR. Something like:
```suggestion
- [CL-2168] Removed support for many old image formats, and added support for webp images. The supported formats are now jpg, jpeg, gif, png, webp, svg.
```